Assets/Scripts3D/Generator3D.cs

```diff
@@ -235,21 +244,41 @@ class Room {
         }
     }
 
-    void PlaceCube(Vector3Int location, Vector3Int size, Material material) {
+    // Floor prefab
+    void PlaceCube(Vector3Int location, Vector3Int size, Material material, string tag) {
         GameObject go = Instantiate(cubePrefab, location, Quaternion.identity);
+        // GameObject go = Instantiate(floorPrefab, location, Quaternion.identity);
         go.GetComponent<Transform>().localScale = size;
         go.GetComponent<MeshRenderer>().material = material;
+        go.tag = tag;
+    }
+
+    // Corridor prefab
+    void PlaceCorridorPrefab(Vector3Int location, Vector3Int size, Material material) {
```

Comment:
```suggestion
    void PlaceCorridorPrefab(Vector3Int location, Vector3Int size, Material material) 
    {
```
Opening brace on newline always